Description: Honor DESTDIR during installation.
Author: Peter Pentchev <roam@ringlet.net>
Forwarded: not yet
Last-Update: 2013-10-09

--- a/spiped/Makefile
+++ b/spiped/Makefile
@@ -8,16 +8,16 @@
 all:	${PROG}
 
 install:${PROG}
-	mkdir -p ${BINDIR}
-	cp ${PROG} ${BINDIR}/_inst.${PROG}.$$$$_ &&	\
-	    strip ${BINDIR}/_inst.${PROG}.$$$$_ &&	\
-	    chmod 0555 ${BINDIR}/_inst.${PROG}.$$$$_ && \
-	    mv -f ${BINDIR}/_inst.${PROG}.$$$$_ ${BINDIR}/${PROG}
+	mkdir -p ${DESTDIR}${BINDIR}
+	cp ${PROG} ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ &&	\
+	    strip ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ &&	\
+	    chmod 0555 ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ && \
+	    mv -f ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ ${DESTDIR}${BINDIR}/${PROG}
 	if ! [ -z "${MAN1DIR}" ]; then			\
 		for MPAGE in ${MAN1}; do						\
-			cp $$MPAGE ${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&			\
-			    chmod 0444 ${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&		\
-			    mv -f ${MAN1DIR}/_inst.$$MPAGE.$$$$_ ${MAN1DIR}/$$MPAGE;	\
+			cp $$MPAGE ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&			\
+			    chmod 0444 ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&		\
+			    mv -f ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ ${DESTDIR}${MAN1DIR}/$$MPAGE;	\
 		done;									\
 	fi
 
--- a/spipe/Makefile
+++ b/spipe/Makefile
@@ -8,16 +8,16 @@
 all:	${PROG}
 
 install:${PROG}
-	mkdir -p ${BINDIR}
-	cp ${PROG} ${BINDIR}/_inst.${PROG}.$$$$_ &&	\
-	    strip ${BINDIR}/_inst.${PROG}.$$$$_ &&	\
-	    chmod 0555 ${BINDIR}/_inst.${PROG}.$$$$_ && \
-	    mv -f ${BINDIR}/_inst.${PROG}.$$$$_ ${BINDIR}/${PROG}
+	mkdir -p ${DESTDIR}${BINDIR}
+	cp ${PROG} ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ &&	\
+	    strip ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ &&	\
+	    chmod 0555 ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ && \
+	    mv -f ${DESTDIR}${BINDIR}/_inst.${PROG}.$$$$_ ${DESTDIR}${BINDIR}/${PROG}
 	if ! [ -z "${MAN1DIR}" ]; then			\
 		for MPAGE in ${MAN1}; do						\
-			cp $$MPAGE ${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&			\
-			    chmod 0444 ${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&		\
-			    mv -f ${MAN1DIR}/_inst.$$MPAGE.$$$$_ ${MAN1DIR}/$$MPAGE;	\
+			cp $$MPAGE ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&			\
+			    chmod 0444 ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ &&		\
+			    mv -f ${DESTDIR}${MAN1DIR}/_inst.$$MPAGE.$$$$_ ${DESTDIR}${MAN1DIR}/$$MPAGE;	\
 		done;									\
 	fi
 
